src/lib/utils/project/GitProjectService.ts

```diff
@@ -408,17 +404,51 @@ export default class GitProjectService {
       };
 
       maintainersSplitsMetadata.push(receiver);
-      receivers.push(receiver);
     }
 
+    const dependenciesAndMaintainers = dependenciesSplitMetadata
+      .map((d) => ({
+        ...d,
+        typeOfReceiver: 'Dependency',
+      }))
+      .concat(
+        maintainersSplitsMetadata.map((m) => ({
+          ...m,
+          typeOfReceiver: 'Maintainer',
+        })),
+      );
+    const roundedReceivers = roundWeights(dependenciesAndMaintainers as any);
+    const roundedDependencies = roundedReceivers
```

Comment:
let's call these "dependenciesMetadataRounded" and "maintainersMetadataRounded"? just a bit more explicit & easier to understand IMO